src/main/java/nextstep/subway/line/domain/Fare.java

```diff
@@ -0,0 +1,41 @@
+package nextstep.subway.line.domain;
+
+import nextstep.subway.line.exception.FareException;
+
+import javax.persistence.Column;
+import javax.persistence.Embeddable;
+
+@Embeddable
+public class Fare {
+    private static final String FARE_MIN_SIZE_MESSAGE = "추가요금은 음수일 수 없습니다.";
+    private static final int FARE_MIN_VALUE = 0;
+    @Column(name = "extraFare")
+    private int value;
```

Comment:
객체로 잘 포장해주셨네요 💯 

`요금과 관련된 검증과 행위`를 담당하도록 해볼 수 있을 것 같아요.
예를들면 `plus()`, `minus()`, `multiply()` 등이 있겠네요.

[불변객체를 만드는 방법](https://tecoble.techcourse.co.kr/post/2020-05-18-immutable-object/)

```suggestion
@Embeddable
public class Fare {
    private static final String FARE_MIN_SIZE_MESSAGE = "추가요금은 음수일 수 없습니다.";
    private static final int FARE_MIN_VALUE = 0;
    
    @Column(name = "extraFare")
    private int value;
```